File: masterclass/Demomvvm/app/src/main/java/com/example/demo_mvvm/home/HomeUiData.kt
package com.example.demo_mvvm.home

import androidx.compose.ui.graphics.Color
import androidx.compose.ui.graphics.vector.ImageVector

data class Transaction(
    val id: String,
    val title: String,
    val category: String,
    val amount: String,
    val time: String,
    val icon: ImageVector,
    val badgeColor: Color,
    val rowColor: Color,
)

data class HomeUiData(
    val userName: String,
    val activityValues: List<Int>,
    val activityLabels: List<String>,
    val salesLastWeek: String,
    val revenueLastWeek: String,
    val transactions: List<Transaction>,
)

File: masterclass/Demo-mvp-iOS/Demo-mvp-iOS/Home/HomeUiData.swift
import UIKit

struct Transaction {
    let id: String
    let title: String
    let category: String
    let amount: String
    let time: String
    let iconSystemName: String
    let badgeColor: UIColor
    let rowBackgroundColor: UIColor
}

struct HomeUiData {
    let userName: String
    let activityValues: [Int]
    let activityLabels: [String]
    let salesLastWeek: String
    let revenueLastWeek: String
    let transactions: [Transaction]
}
